File: BookStore.BuisneLogic/Models/Users/UserModelItem.cs
using BookStore.BusinessLogic.Models.Base;

namespace BookStore.BusinessLogic.Models.Users
{
    public class UserModelItem : BaseModel
    {
        public long Id { get; set; }
        public string UserName { get; set; }
        public string NewPassword { get; set; }
        public string CurrentPassword { get; set; }
        public string Role { get; set; }
        public string Email { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Photo { get; set; }
        public bool Status { get; set; }
    }
}

File: StoreClient/src/app/shared/models/Users/user-model-item.ts
import { BaseModel } from '../Base/base-model';

export class UserModelItem extends BaseModel {
  id: number;
  userName: string;
  email: string;
  status: boolean;
  firstName: string;
  lastName: string;
  newPassword: string;
  currentPassword: string;
  role: string;
  photo: string;
}
